test-worker/test/asset-compute/worker/png-background-color/params.json

```diff
@@ -0,0 +1,4 @@
+{
+    "fmt": "png",
+    "pdfbgcolor": "#0EF"
```

Comment:
This tests mimics adding a color as a background to any image with a transparent background